content/800-data-platform/050-data-proxy/700-prisma-cli-with-data-proxy.mdx

```diff
@@ -9,19 +9,17 @@ tocDepth: 3
 
 Currently, you can only use [Prisma Client](/concepts/components/prisma-client) with the Data Proxy to _query_ your database.
 
-</TopBlock>
-
+You need a direct (non-Data Proxy) database connection if you apply schema changes to your database with [Prisma Migrate](/concepts/components/prisma-migrate/get-started) or [`db push`](/concepts/components/prisma-migrate/db-push), or if you [introspect](/concepts/components/introspection) your database. See [Prisma CLI commands that require a direct database connection](#prisma-cli-commands-that-require-a-direct-database-connection).
 
-### Problem
-If you want to run [migrations](/concepts/components/prisma-migrate/get-started), [introspection](/concepts/components/introspection),  or any of the [Prisma CLI commands](/reference/api-reference/command-reference) listed below, you can only do so on a direct (non-Data Proxy) database connection.
+In Prisma versions 4.9.0 and later, you can [set a direct database connection URL in your Prisma schema](#set-a-direct-database-connection-url-in-your-prisma-schema).
 
-<Admonition type="warning">
+In earlier versions, you must add an [environment variable](/guides/development-environment/environment-variables) with the direct database connection URL and overwrite the [`DATABASE_URL` environment variable](/guides/development-environment/environment-variables#example-set-the-database_url-environment-variable-in-an-env-file) when you need a direct connection.
```

Comment:
This `#example-set-the-database_url-environment-variable-in-an-env-file` is not valid anymore, since the content is removed by this PR